egs/aurora4/asr1/run.sh

```diff
@@ -0,0 +1,282 @@
+#!/bin/bash
+# Copyright 2019 Johns Hopkins University (Ziyan Jiang)
+#  Apache 2.0  (http://www.apache.org/licenses/LICENSE-2.0)
+
+. ./path.sh
+. ./cmd.sh
+
+# general configuration
+backend=pytorch
+stage=0        # start from 0 if you need to start from data preparation
+stop_stage=100
+ngpu=0         # number of gpus ("0" uses cpu, otherwise use gpu)
+debugmode=1
+dumpdir=dump   # directory to dump full features
+N=0            # number of minibatches to be used (mainly for debugging). "0" uses all minibatches.
+verbose=0      # verbose option
+resume=        # Resume the training from snapshot
+seed=1
+
+# feature configuration
+do_delta=false
+
+train_config=conf/train.yaml
+lm_config=conf/lm.yaml
+decode_config=conf/decode.yaml
+
+# rnnlm related
+use_wordlm=true     # false means to train/use a character LM
+lm_vocabsize=65000  # effective only for word LMs
+lm_resume=          # specify a snapshot file to resume LM training
+lmtag=              # tag for managing LMs
+
+# decoding parameter
+recog_model=model.acc.best # set a model to be used for decoding: 'model.acc.best' or 'model.loss.best'
+
+# data
+aurora4=/export/corpora5/AURORA
+wsj0=/export/corpora5/LDC/LDC93S6B
+wsj1=/export/corpora5/LDC/LDC94S13B
+
+# exp tag
+tag="" # tag for managing experiments.
+
+. utils/parse_options.sh || exit 1;
+
+. ./path.sh
+. ./cmd.sh
+
+# Set bash to 'debug' mode, it will exit on :
+# -e 'error', -u 'undefined variable', -o ... 'error in pipeline', -x 'print commands',
+set -e
+set -u
+set -o pipefail
+
+train_set=train_mix
```

Comment:
Could you please add some comments about `train_mix` and the original `train_si84_multi` (this is the AURORA4 standard training data, right?) here?
Then, the user could select `train_mix` based custom training set or `train_si84_multi` based original set to report their numbers.

Reply:
Yes, I have added comments about the training dataset.